src/rdf4cpp/rdf/datatypes/owl/Rational.hpp

```diff
@@ -0,0 +1,46 @@
+#ifndef RDF4CPP_OWL_RATIONAL_HPP
+#define RDF4CPP_OWL_RATIONAL_HPP
+
+#include <rdf4cpp/rdf/datatypes/registry/DatatypeMapping.hpp>
+#include <rdf4cpp/rdf/datatypes/registry/LiteralDatatypeImpl.hpp>
+
+#include <boost/multiprecision/cpp_int.hpp>
+
+namespace rdf4cpp::rdf::datatypes::registry {
+
+inline constexpr util::ConstexprString owl_rational{"http://www.w3.org/2002/07/owl#rational"};
+
+template<>
+struct DatatypeMapping<owl_rational> {
+    using cpp_datatype = boost::multiprecision::cpp_rational;
+};
+
+template<>
+capabilities::Default<owl_rational>::cpp_type capabilities::Default<owl_rational>::from_string(std::string_view s);
+
+template<>
+std::string capabilities::Default<owl_rational>::to_string(cpp_type const &value);
```

Comment:
could the `capabilities::Default<T>::to_string` be made noexcept?

Reply:
yes I think so, added it to #84